airgun/views/contenthost.py

```diff
@@ -78,6 +78,18 @@ def read(self):
         return self.color
 
 
+class UnEvenActionDropDown(ActionsDropdown):
```

Comment:
Think the name is ambiguous here
```
class ModuleStreamActionDropDown(ActionsDropdown):
    ...
```
Note the name is singular, as it's for one Module steam row